reascripts/ReaSpeech/source/CurlRequest.lua

```diff
@@ -292,21 +310,26 @@ function CurlRequest:curl_timeout_argument()
 end
 
 function CurlRequest:check_sentinel()
-  local sentinel = io.open(self.sentinel_file, 'r')
+  local sentinel = io.open(self.progress_file, 'r')
 
   if not sentinel then
     return false
   end
 
-  local contents = sentinel:read("a")
+  local contents = sentinel:read("*all")
   sentinel:close()
 
-  return contents and contents == "done"
+  -- Workaround for https://github.com/curl/curl/issues/10491
+  local version = CurlRequest.curl_version()
+  if version and version[1] < 8 then return true end
+
+  self:debug('Checking sentinel: ' .. contents)
+  return contents and contents:find(CurlRequest.SENTINEL)
 end
 
 function CurlRequest:http_status_and_body(response)
   local headers, content = CurlRequest._split_curl_response(response)
-  self:debug('Parsing response: ' .. dump(headers) .. ', ' .. dump(content))
+  -- self:debug('Parsing response: ' .. dump(headers) .. ', ' .. dump(content))
```

Comment:
assuming these are left in but commented out because it's a very conceivable future where we're looking at this debug output

Reply:
Yeah, it's useful debug output when working on this part of the code, but it is also verbose enough to cause other log messages to become inaccessible due to the Reaper console character limit.